src/elementary/ParticlePairID.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef NJOY_ELEMENTARY_PARTICLEPAIRID
+#define NJOY_ELEMENTARY_PARTICLEPAIRID
+
+// system includes
+#include <string>
+
+// other includes
+#include "elementary/src/split.hpp"
+#include "elementary/ParticleID.hpp"
+#include "elementary/Identifier.hpp"
+
+namespace njoy {
+namespace elementary {
+
+  /**
+   *  @class
+   *  @brief The identifier for the special case of a particle pair
+   */
+  class ParticlePairID : public Identifier< ParticlePairID > {

```

Comment:
This seems a bit overkill when I think you could just use a c++ alias.

Reply:
I can't use an alias if I want to restrict the constructor to two particles. I tried reusing the tuple identifier but that didn't really work out as nicely as I had hoped so I went with the Identifier instead.